VERSION ""


NS_ : 
	NS_DESC_
	CM_
	BA_DEF_
	BA_
	VAL_
	CAT_DEF_
	CAT_
	FILTER
	BA_DEF_DEF_
	EV_DATA_
	ENVVAR_DATA_
	SGTYPE_
	SGTYPE_VAL_
	BA_DEF_SGTYPE_
	BA_SGTYPE_
	SIG_TYPE_REF_
	VAL_TABLE_
	SIG_GROUP_
	SIG_VALTYPE_
	SIGTYPE_VALTYPE_
	BO_TX_BU_
	BA_DEF_REL_
	BA_REL_
	BA_DEF_DEF_REL_
	BU_SG_REL_
	BU_EV_REL_
	BU_BO_REL_
	SG_MUL_VAL_

BS_:

BU_:


BO_ 802 LDWS: 8 Vector__XXX
 SG_ state_of_LDWS : 0|4@1+ (1,0) [0|3] "" Vector__XXX

BO_ 801 FCW: 8 Vector__XXX
 SG_ time_to_collision : 8|8@1+ (0.1,0) [0|10] "" Vector__XXX
 SG_ object_type : 4|4@1+ (1,0) [0|2] "" Vector__XXX
 SG_ state_of_FCW : 0|4@1+ (1,0) [0|3] "" Vector__XXX

BO_ 800 DSM: 8 Vector__XXX
 SG_ state_of_DSM : 0|4@1+ (1,0) [0|3] "" Vector__XXX

BO_ 703 CLU_info: 8 Vector__XXX
 SG_ vehicle_speed_high_precision : 48|16@1+ (0.01,0) [0|150] "" Vector__XXX
 SG_ steering_angle : 32|16@1- (0.1,0) [-3276.8|3276.7] "" Vector__XXX
 SG_ vehicle_speed_low_precision : 16|8@1+ (1,0) [0|255] "" Vector__XXX
 SG_ turn_signal_right_indicate : 9|1@1+ (1,0) [0|1] "" Vector__XXX
 SG_ turn_signal_left_indicate : 8|1@1+ (1,0) [0|1] "" Vector__XXX
 SG_ clu_ready : 0|4@1+ (1,0) [0|7] "" Vector__XXX



BA_DEF_  "MultiplexExtEnabled" ENUM  "No","Yes";
BA_DEF_  "BusType" STRING ;
BA_DEF_DEF_  "MultiplexExtEnabled" "No";
BA_DEF_DEF_  "BusType" "";
BA_ "BusType" "CAN";

